frontend/src/models/round.js

```diff
@@ -25,7 +25,9 @@ export class Round {
   }
 
   nextTrick() {
-    return new Trick(this.players.length);
+    let trick = new Trick(this.players.length);
+    if (this.cardsLeft() <= 4) trick.setLastTrickInRound();
```

Comment:
can we change this to `if (this.cardsLeft() <= this.players.length)` just for the sake of removing the magic number? :slightly_smiling_face:

Reply:
Good call, I don't like static numbers either, just didn't think of it 👍